cmd/access/node_builder/access_node_builder.go

```diff
@@ -84,7 +84,9 @@ type AccessNodeBuilder interface {
 // while for a node running as a library, the config fields are expected to be initialized by the caller.
 type AccessNodeConfig struct {
 	staked                       bool
-	stakedAccessNodeIDHex        string
+	bootstrapNodeAddresses       []string
```

Comment:
now the unstaked node only needs the bootstrap node addresses and the public keys of those nodes.

Reply:
I don't think that is an improvement at all. Now, we have to manage multiple node identities _with_ public keys on the unstaked consensus follower side. That doesn't make sense. Knowing a single access node address should be the only dependency needed. The DHT information should not be information that you need to connect; it should be served by the access node we connect to.